### ISSUE TITLE
Node 12 not existing in FreeBSD 13 => Haraka fails to provision

### ISSUE THREAD

**Issue by Infern1** [msimerson/Mail-Toaster-6#488]
**Describe the bug**
provision new Haraka jail fails due to node 12 dependency

**Comment by msimerson** [comment]
I'm aware. On my todo list is to test against node 14. It should work fine and I suggest giving it a whirl. The easy way to do that is edit ./provision/haraka.sh, edit the build file, and then run `sh ./provision/haraka.sh`

**Comment by Infern1** [comment]
Yes I know howto change, however already late, so skipped it. Just opened an issue to not forget.

Also fighting now with the new Haproxy 2.4.5 so first solve those issues
